lib/template/macros.c

```diff
@@ -190,6 +190,9 @@ LogMacroDef macros[] =
   { "SDATA", M_SDATA },
   { "MSGHDR", M_MSGHDR },
   { "SOURCEIP", M_SOURCE_IP },
+  { "DESTIP", M_DEST_IP },
+  { "DESTPORT", M_DEST_PORT },
+  { "PROTO", M_PROTO },
```

Comment:
Checked an example message with an `stdin` source, and these macros are filled,
```
"DESTPORT":"0"
"DESTIP":"127.0.0.1"
"PROTO":"0"
```
though these are network source specific macros.

Maybe we can reason for `DESTIP`, the way `SOURCEIP` exists.
`DESTPORT` and `PROTO` still looks strange. On the upside, zero for port is not valid value.
On the other hand, the zero value for proto is a valid protocol number for `HOPOPT`.

I was thinking if we would be better off with empty values, but that can also create confusion: users need to use `--omit-empty-values` (once that PR is merged) to exclude these values. Which is not easy remember to look out for.